deploy/docker/cp-edge/wetty/app.js

```diff
@@ -317,22 +332,26 @@ io.on('connection', function(socket) {
             run_ssh_mode = get_boolean_preference('system.ssh.default.root.user.enabled', auth_key) ? 'root' : 'owner';
         }
     }
+    const run_user_name_case = run_details.parameters['CP_CAP_USER_NAME_CASE'] || 'default';
+    const run_user_name_metadata_key = run_details.parameters['CP_CAP_USER_NAME_METADATA_KEY'] || 'local_user_name';
     const current_user = get_current_user(auth_key);
+    const current_user_metadata = current_user ? get_user_metadata(current_user.id, auth_key) : {};
+    const current_user_login_name = current_user ? current_user.name : undefined;
+    const current_user_local_name = current_user_metadata[run_user_name_metadata_key];
+    const current_user_name = current_user_local_name
```

Comment:
if we came up with `undefined` wouldn't it be better to fail with an error into `wetty` console? rather then try to login

Reply:
You are definitely right. Decided to introduce a general error handling with a user level errors in the future.